src/installLogsPrinter.js

```diff
@@ -57,6 +60,18 @@ function installLogsPrinter(on, options = {}) {
   }
 }
 
+function logOutputTarget(processor) {
+  let message;
+  if (processor instanceof OUTPUT_PROCESSOR_TYPE.txt) {
```

Comment:
Let's make it so that when adding new output processor type it will still function properly: `if (Object.values(OUTPUT_PROCESSOR_TYPE).find((type) => processor instanceof type)`

Reply:
done